### DISCUSSION TITLE
Vim 특강 | 2022.10.19

### DISCUSSION THREAD

**Discussion by leftykhim** [woowacourse/retrospective#28] [General]
# 함께 이야기 하기
특강 내용을 다시 돌아보며 내가 평소에 사용하는 에디터에도 적용할 수 있을지 페어와 함께 이야기해 봅시다.

✅ 오늘 특강에서 사용했던 기능을 페어와 함께 정리해 보세요.
✅ 특강에서 사용했던 기능 중 내가 평소에 사용하는 에디터에서도 가능한 기능을 최소 3개 이상 찾아보세요.
✅ 오늘 배우지 않은 기능 중 내가 평소에 사용하는 에디터에서 가능한 기능을 최소 3개 이상 작성해 보세요.
✅ 다른 크루에게 공유하고 싶은 나만의 에디터 꿀팁이 있다면?

# 역할
- 이끔이: 닉네임을 오름차순으로 정렬했을 때 가장 빠른 크루. 대화를 이끈다.
- 기록이: 그다음으로 빠른 크루. 이야기 나눈 내용을 기록해 공유한다.
- 시간이: 나머지 크루. 활동이 시간 내에 끝나도록 시간을 관리한다.

# 템플릿
```
### 특강에서 사용했던 기능 중 내가 평소에 사용하는 에디터에서도 가능한 기능 (최소 3개 이상)
### 오늘 배우지 않은 기능 중 내가 평소에 사용하는 에디터에서 가능한 기능 (최소 3개 이상)
### 다른 크루에게 공유하고 싶은 나만의 에디터 꿀팁
### 오늘 Vim 특강에 대한 피드백 부탁해요.
```

**Comment by skullkim** [comment]
너무 좋아요😍

**Comment by yaho99** [comment]
재미있었어요~~! 잘 몰라서 어렵지만 익숙해지면 엄청 유용할 것 같아요 😆

**Comment by airman5573** [comment]
유익한 수업 감사합니다.  
여럽지만 연습해보겠습니다 :D

**Comment by InKyoJeong** [comment]
유익한 수업이였어요. vim을 좀더 잘 활용할수 있게 됐어요!

**Comment by Youngyoon-1** [comment]
매크로 기능 대박이네요!! 수업 정말 재미있었습니다!! 감사합니당 👍
